### ISSUE TITLE
Guided Sampling-based Evolutionary Deep Neural Network for Intelligent Fault Diagnosis. (arXiv:2111.06885v2 [cs.NE] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#16045]
Guided Sampling-based Evolutionary Deep Neural Network for Intelligent Fault Diagnosis. (arXiv:2111.06885v2 [cs.NE] UPDATED)<br>
https://ift.tt/S8gjE14<br>
<p>The diagnostic performance of most of the deep learning models is greatly affected by the selection of model architecture and hyperparameters. Manual selection of model architecture is not feasible as training and evaluating the different architectures of deep learning models is a time-consuming process. Therefore, we have proposed a novel framework of evolutionary deep neural network which uses policy gradient to guide the evolution of DNN architecture towards maximum diagnostic accuracy. We have formulated a policy gradient-based controller which generates an action to sample the new model architecture at every generation such that the optimality is obtained quickly. The fitness of the best model obtained is used as a reward to update the policy parameters. Also, the best model obtained is transferred to the next generation for quick model evaluation in the NSGA-II evolutionary framework. Thus, the algorithm gets the benefits of fast non-dominated sorting as well as quick model evaluation. The effectiveness of the proposed framework has been validated on three datasets: the Air Compressor dataset, Case Western Reserve University dataset, and Paderborn university dataset.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
